aspnetcore/fundamentals/environments.md

```diff
@@ -89,6 +89,14 @@ These commands take effect only for the current window. When the window is close
 
 ![ASPNET Core Environment Variable](environments/_static/windows_aspnetcore_environment.png) 
 
+**web.config**
+
+See [Setting environment variables in the ASP.NET Core Module configuration reference](xref:hosting/aspnet-core-module#setting-environment-variables) topic.
```

Comment:
Just a nit that, feel free to ignore, but if you're going to have all that in link text, I think "topic" should be included as well.